src/e2e/iOS/helpers/getEditingText.ts

```diff
@@ -1,9 +1,10 @@
 import { Browser } from 'webdriverio'
+import { editable } from '../../../../styled-system/recipes'
 
 /** Get the thought value that cursor on. */
 const getEditingText = (browser: Browser<'async'>): Promise<string | undefined> => {
   return browser.execute(() => {
-    return document.querySelector('.editing .editable')?.innerHTML
+    return document.querySelector(`[data-editing=true] ${editable()}`)?.innerHTML
```

Comment:
I think this should be `[data-editing=true] .${editable()}` (added: `.`)

I'm curious if this will work inside the webdriver's browser context.